poll/public/handlebars/poll_studio.handlebars

```diff
@@ -0,0 +1,22 @@
+<script id="poll-form-component" type="text/html">
+{{#each items}}
+<li class="field comp-setting-entry is-set poll-{{noun}}-studio-item">
+    <div class="wrapper-comp-setting">
+        <label class="label setting-label poll-setting-label" for="{{noun}}-label-{{key}}">{{noun}}</label>
+        <input class="input setting-input" name="{{noun}}-label-{{key}}" id="{{noun}}-label-{{key}}" value="{{text}}" type="text" /><br />
+        {{#if image}}
+        <label class="label setting-label" for="{{noun}}-img-{{key}}">Image URL</label>
+        <input class="input setting-input" name="{{noun}}-img-{{key}}" id="{{noun}}-img-{{key}}" value="{{img}}" type="text" />
+        {{/if}}
+        <div class="poll-move">
+            <div class="poll-move-up">&#9650;</div>
```

Comment:
This control is inaccessible (```<div>``` elements have no accessible roles, states or properties associated with them)  The user cannot move focus with the keyboard, access an accessible name, or otherwise interact with this control at all.  Recommend the following:
```<button class="poll-move-up">&#9650;<span class="sr">&nbsp;move poll up</span></button>```
Button elements come with all sorts of cool behaviors and properties for free, like you can focus on them, click them, get their accessible name, etc.  Same applies for the move down control below.